packages/core/src/browser/shell-command-contribution.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { inject, injectable } from 'inversify'
+import { Command, CommandRegistry, CommandContribution } from '../common';
+import { MenuContribution, MenuModelRegistry } from '../common/menu';
+import { ApplicationShell, MAINAREA_TABBAR_CONTEXT_MENU } from './shell';
+
+export namespace ShellCommands {
```

Comment:
We have `CommonCommands` already, some of them as `NEXT_TAB` as well based on the shell. I am not sure whether it is good to have several command contributions in the core. It could be hard to discover for extension developers.

Reply:
OK I'll move them there and user the core.* prefix